lib/pages/my_reports_pages/components/reports_list_widget.dart

```diff
@@ -12,6 +12,21 @@ class ReportsList extends StatelessWidget {
 
   ReportsList(this.reports, this.onTap);
 
+  String getTranslatedReportType(BuildContext context, String? reportType){
```

Comment:
Does `reportType` can be null in the code?

Reply:
Right now it can theoretically be null, as defined in the Reports class. That's because after flutter 3.0.0 all types require null safety.

Maybe it's better to change the nomenclature to this so that it can't be null, but if we decide to do this, then we have to handle it correctly for this variable and also the others which are nullable right now
```
class Report {
  // other vars
  late String type;  // before: String? type;

  Report(
      // other vars
      required this.type, // before: this.type,
```

Reply:
In the backend `Report.type` is a mandatory field (see: https://github.com/Mosquito-Alert/mosquito_alert/blob/e0618655f6320c9befb29b0741a35fa096fe8c64/tigaserver_app/models.py#L416) so it would make sense to make it required in the app code too. If you think we might address this in a new issue we can create a one for checking the required variables in the code.

Reply:
> If you think we might address this in a new issue we can create a one for checking the required variables in the code.

Yeah I think it makes sense to address all or most in a dedicated pull request